apps/frontend/src/components/cards/StatusCardRow.vue

```diff
@@ -101,44 +110,44 @@ export default class StatusCardRow extends Vue {
         icon: 'check-circle',
         title: 'Passed',
         subtitle: `${StatusCountModule.countOf(
-          this.filter,
+          {fromFile: this.filter.fromFile},
           'PassedTests'
         )} individual checks passed`,
         color: 'statusPassed',
-        number: StatusCountModule.countOf(this.filter, 'Passed')
+        number: StatusCountModule.countOf({fromFile: this.filter.fromFile, omit_overlayed_controls: this.filter.omit_overlayed_controls}, 'Passed')
       },
       {
         icon: 'close-circle',
         title: 'Failed',
         subtitle: `${StatusCountModule.countOf(
-          this.filter,
+          {fromFile: this.filter.fromFile},
           'PassingTestsFailedControl'
         )} individual checks passed, ${StatusCountModule.countOf(
-          this.filter,
+          {fromFile: this.filter.fromFile},
           'FailedTests'
         )} failed out of ${StatusCountModule.countOf(
-          this.filter,
+          {fromFile: this.filter.fromFile},
           'PassingTestsFailedControl'
         ) + StatusCountModule.countOf(
-          this.filter,
+          {fromFile: this.filter.fromFile},
```

Comment:
Was `omit_overlayed_controls` passed in here previously? Should it still be passed in now? Can you just build a object with the data you need from `this.filter` and pass that to all of these instead of creating slightly different objects for each?